model/aspnetcore/metrics.yaml

```
@@ -135,3 +141,298 @@ groups:
     attributes:
       - ref: aspnetcore.rate_limiting.result
         requirement_level: required
+
+  # identity
+  - id: metric.aspnetcore.identity.user.create
+    type: metric
+    metric_name: aspnetcore.identity.user.create
+    annotations:
+      code_generation:
+        metric_value_type: int
+    stability: development
+    brief: The number of users created.
+    instrument: counter
+    unit: "{user}"
+    note: |
+      Meter name: `Microsoft.AspNetCore.Identity`; Added in: ASP.NET Core 10.0
+    extends: aspnetcore.common.identity.metrics.attributes
+    attributes:
+      - ref: aspnetcore.identity.result
+        requirement_level:
+          conditionally_required: if no exception was thrown.
+      - ref: aspnetcore.identity.error_code
+        requirement_level:
+          conditionally_required: if an error was set on a failed identity result.
+      - ref: error.type
+        brief: The full name of exception type or the identity error code.
+        requirement_level:
+          conditionally_required: if and only if an error has occurred.
+        examples: ['System.OperationCanceledException', 'PasswordMismatch']
```

to override generic note

```suggestion
        examples: ['System.OperationCanceledException', 'PasswordMismatch']
        note: ""
```

(also on other metrics)